conans/model/conan_file.py

```diff
@@ -1,7 +1,7 @@
 import os
 from contextlib import contextmanager
 
-from conans import tools  # @UnusedImport KEEP THIS! Needed for pyinstaller to copy to exe.
+# TODO: from conans import tools  # @UnusedImport KEEP THIS! Needed for pyinstaller to copy to exe.
```

Comment:
Maybe this is needed for pyinstaller to "copy" the tools import to the bundle, otherwise people using the standalone installer might run into issues when using from `conans import tools` in their recipes. But not sure, will have to be tested

Reply:
I make this comment just to be sure that we check if this line is still required.

Reply:
It was required for sure. Be careful, because removing it, all tests pass, but it is later in the pyinstaller, which has missing modules.